config/locales/api/en.yml

```diff
@@ -19,6 +19,22 @@ en:
     patients_enrolled: "How many new patients are enrolled at %{facility_name}?"
     all_time_patients: "All time: %{count} patients"
     updated_at: "Updated at %{time}"
+  user_analytics:
+    hypertension: 'Hypertension'
+    diabetes: 'Diabetes'
+    genders: ['all', male', 'female', 'transgender']
+    all_time: 'All-time'
+    follow_up_patients: 'Follow-up hypertension patients'
+    registered_patients: 'Registered Patients'
+    hypertension_control: 'Hypertension control'
+    hypertension_control_desc: 'How many patients with hypertension had BP below 140/90 at their last follow-up visit in the month?'
+    notes: 'Notes'
+    footer_note_1: "Data is for all activity at %{facility_name}. If data appears out of date, tap the \"Sync pending\" link on the home screen."
```

Comment:
Easier to wrap this in single quotes? I think the interpolation still works right?

Reply:
Resolved. It works!